packages/atlas/src/views/viewer/PlaylistView/PlaylistDetails.tsx

```diff
@@ -0,0 +1,66 @@
+import { SvgActionLinkUrl, SvgActionPlay } from '@/assets/icons'
+import { Text } from '@/components/Text'
+import { Button } from '@/components/_buttons/Button'
+import { ChannelLink } from '@/components/_channel/ChannelLink'
+import { ColumnGapBlock, RowGapBlock } from '@/components/_layouts'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+
+import {
+  Counter,
+  CustomVideoListItemWrapper,
+  DetailsButtonsWrapper,
+  DetailsWrapper,
+  InfoContainer,
+  StyledVideoListItem,
+  Thumbnail,
+  Wrapper,
+} from './PlaylistDetails.styles'
+
+export const PlaylistDetails = () => {
+  const xsMatch = useMediaMatch('xs')
+  const smMatch = useMediaMatch('sm')
+
+  return (
+    <Wrapper>
+      <InfoContainer>
+        <DetailsWrapper>
+          <Thumbnail src="https://picsum.photos/seed/picsum/400/200" />
+          <RowGapBlock gap={2}>
+            <Text variant="h500" as="p">
+              Random stuff playlist
+            </Text>
+            <Text variant="t200" as="p" color="colorTextMuted">
+              7 videos • 1:23:42 total • Last update: 24 Sept 2021
+            </Text>
+            <Text variant="t200" as="p" color="colorTextMuted">
+              There goes some description of the playlist. Can be like 2, maybe 4 rows big. Maybe a bit bigger? Let me
+              know! Lorem ipsum dolor sit amet, lorem ipsum dolor sit amet{' '}
+            </Text>
+            <DetailsButtonsWrapper align="center" justify="space-between">
+              <Button icon={<SvgActionPlay />} fullWidth={!xsMatch}>
+                Play all videos
+              </Button>
+              <Button icon={<SvgActionLinkUrl />} fullWidth={!xsMatch} variant="tertiary">
+                Copy link
+              </Button>
+            </DetailsButtonsWrapper>
+          </RowGapBlock>
+        </DetailsWrapper>
+
+        <ColumnGapBlock justify="space-between" padding={6}>
+          <ChannelLink id="1" followButton />
+        </ColumnGapBlock>
+      </InfoContainer>
+      <RowGapBlock>
+        {Array(10)
+          .fill(1)
+          .map((_, index) => (
+            <CustomVideoListItemWrapper key={index}>
+              <Counter>{index + 1}</Counter>
+              <StyledVideoListItem id={String(index)} variant={xsMatch && !smMatch ? 'medium' : 'large'} />
```

Comment:
Doesn't match the designs:
![image](https://user-images.githubusercontent.com/51168865/213186361-c33b7ffd-8030-4c0d-a6d3-55e9a6b82945.png)
![image](https://user-images.githubusercontent.com/51168865/213187102-60340f6e-c548-499b-9d09-4fcc1b021a10.png)

Check the position of the title and description.
Also, I think we shouldn't use here the ListItem, we should use the VideoTile component here. I think we already have a horizontal variant implemented if I'm not mistaken.
https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-components?node-id=3630%3A118313&t=DnHdi9dR8x1yog5s-4
We shouldn't have a hover effect in this particular case as well.